compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -35,13 +35,14 @@ class CircleOptimizer final
       FuseBCQ,
       FuseInstanceNorm,
       ResolveCustomOpBatchMatMul,
-      QuantizeWithMinMax,
+      QuantizeDequantizeWeights,
+      QuantizeWithMinMax
```

Comment:
```suggestion
      QuantizeDequantizeWeights,
      QuantizeWithMinMax,
```
(trivial) You can prevent unnecessary change then.

Reply:
Changed. Thanks.